spec/javascripts/modules/sticky-element-container.spec.js

```diff
@@ -2,62 +2,72 @@ describe('A sticky-element-container module', function () {
   'use strict'
 
   var GOVUK = window.GOVUK
-  var $ = window.$
-  var instance
 
-  beforeEach(function () {
-    instance = new GOVUK.Modules.StickyElementContainer()
-  })
+  describe('on desktop', function () {
+    var $element
+    var $footer
+    var instance
+
+    beforeEach(function () {
+      $element = $(
+        '<div data-module="sticky-element-container" style="height: 9001px; margin-bottom: 1000px">' +
+          '<div data-sticky-element>' +
+            '<span>Content</span>' +
+          '</div>' +
+        '</div>'
+      )
+
+      instance = new GOVUK.Modules.StickyElementContainer($element[0])
```

Comment:
Interesting that these tests don't appear to need to call the `init` function, yet presumably are still passing? That seems odd.

Reply:
The original code called the `start` function (line 33). Can we add the init function into this test? I'm concerned that if we're not calling it we're not testing it properly somehow.